roxie/ccd/ccdprotocol.cpp

```diff
@@ -2160,6 +2161,15 @@ class RoxieSocketWorker : public ProtocolQueryWorker
                         VStringBuffer xml(" <wuid>%s</wuid>\n", statsWuid.str());
                         response.flushXML(xml, true);
                     }
+                    if (queryPT->getPropBool("@summaryStats", alwaysSendSummaryStats))
```

Comment:
who sets summartStats prop ?

Reply:
Someone that wants to get the summary stats back (the client will set it in the query XML). We could consider adding a checkbox to the eclwatch page for test queries... but that would be a separate Jira